src/__tests__/Schema.test.ts

```diff
@@ -388,6 +388,67 @@ describe('transformPathsToDbPositions', () => {
 	});
 });
 
+describe('transformPathsToOrdinalPositions', () => {
+	const embeddedDefinition = {
+		innerEmbeddedProp: { type: 'string', path: '9' },
+	} satisfies SchemaDefinition;
+	const documentArrayDefinition = {
+		docStringProp: { type: 'string', path: '12' },
+		docNumberProp: { type: 'number', path: '13' },
+	} satisfies SchemaDefinition;
+	const definition = {
+		stringProp: { type: 'string', path: '1' },
+		stringValueProp: { type: 'string', path: '14.2' },
+		stringSubvalueProp: { type: 'string', path: '14.2.3' },
+		stringArrayValueProp: [{ type: 'string', path: '15.2' }],
+		numberProp: { type: 'number', path: '2' },
+		booleanProp: { type: 'boolean', path: '3' },
+		isoCalendarDateProp: { type: 'ISOCalendarDate', path: '4' },
+		isoTimeProp: { type: 'ISOTime', path: '5' },
+		isoCalendarDateTimeProp: { type: 'ISOCalendarDateTime', path: '6' },
+		arrayProp: [{ type: 'string', path: '7' }],
+		nestedArrayProp: [[{ type: 'string', path: '8' }]],
+		embeddedProp: new Schema(embeddedDefinition),
+		documentArrayProp: [
+			{
+				docStringProp: { type: 'string', path: '10' },
+				docNumberProp: { type: 'number', path: '11' },
+			},
+		],
+		documentArraySchemaProp: [new Schema(documentArrayDefinition)],
+	} satisfies SchemaDefinition;
+	const schema = new Schema(definition);
+
+	test('should throw error if path list is empty', () => {
```

Comment:
This test description doesn't seem to align with the test.

Reply:
done. Thank you.